.github/workflows/clean-cache.yml

```diff
@@ -0,0 +1,29 @@
+name: cleanup caches by a branch
```

Comment:
from https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows#force-deleting-cache-entries